src/components/navigation/NavPills/NavPillsStyles.ts

```diff
@@ -0,0 +1,52 @@
+import MuiTab from '@mui/material/Tab'
+import MuiTabs from '@mui/material/Tabs'
+import { styled } from '@mui/material/styles'
+import { includes } from 'ramda'
+
+const localTabsOverriddenProps = ['indicatorColor', 'color', 'selectedColor']
+export const Tabs = styled(MuiTabs, { shouldForwardProp: prop => !includes(prop, localTabsOverriddenProps) })(
+  ({ theme, color, indicatorColor }: any) => {
```

Comment:
we should specify the types here
you can eather define them inline like `{ theme, color, indicatorColor } : { theme: Theme, color: Color , ...etc }`
or define a new type like so
```
type StyledProps {
theme: Theme
color: Color
...
}
```